product_listing.css

```diff
@@ -0,0 +1,108 @@
+@import '/index.css';
+
+:root {
+  --slider-color: rgba(0, 0, 0, 0.418);
+  --wishlist-active: rgb(255, 38, 38);
+}
+
+.body {
+  margin-top: 6rem;
+  height: 100vh;
+  width: 100vw;
+  display: grid;
+  grid-template-columns: 250px 1fr;
```

Comment:
you could probably use rem here

Reply:
i am kind of going for that kind of look, converted the unit to rem